SetupScript.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+###base script to fresh install systems onto base image for raspberry pi
+
+#configure displays
+cd ~
+git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git
+cd Epsilon-Raspberry
+cd primary/
+sudo cp xorg.conf /etc/X11/
+sudo cp config.txt /boot/
+#install rabbit-mq
+cd ~
+git clone https://github.com/UCSolarCarTeam/Epsilon-Dashboard.git
```

Comment:
@bill-luu this was in another step wasn't it?

Reply:
Yes! Lets keep the installation of Epsilon-Dashboard & other programs in a separate install script.

For rabbitmq setup, you can just copy/paste the necessary stuff from EpsilonDashboard.sh